lib/response-generator/index.js

```diff
@@ -2,13 +2,16 @@
 
 class ResponseGenerator {
 
-	static generate(schemaResponse) {
+	static generate(schemaResponse, preferredExampleName) {
 
 		if(schemaResponse.example)
 			return schemaResponse.example;
 
-		if(schemaResponse.examples && schemaResponse.examples.length)
+		if(schemaResponse.examples && Object.values(schemaResponse.examples).length) {
```

Comment:
In all of the examples I saw, examples was in fact the equivalent of an object, not an array. I think this functionality may have been broken before. We still select the first in the case of no match, but this is a bit less meaningful. Objects don't necessarily guarantee order its true, however in practice, the order of objects hasn't changed unexpectedly on me since the days of cursed IE versions.

Reply:
Good catch. This was in fact broken.

I'd just add validation to the examples property in [`lib/components/responses/structs.js`](https://github.com/jormaechea/open-api-mocker/blob/master/lib/components/responses/structs.js#L21) just to be sure that `Object.values()` doesn't break or brings weird behaviours at this point.